### PR TITLE
Add configuration for account identity event consumers

### PR DESCRIPTION
**Pull request by martskins**

### CODE REVIEW

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-cbc
Path: dev-aws/kafka-shared-msk/cbc
Commit ID: 903d4935b84dc852c0218d77af1273828f06cb60
Requested At: 2024-10-09T14:10:28Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/cbc` as comment.

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: prod-aws
Module: pubsub/kafka-shared-msk-cbc
Path: prod-aws/kafka-shared-msk/cbc
Commit ID: 903d4935b84dc852c0218d77af1273828f06cb60
Requested At: 2024-10-09T14:10:29Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan prod-aws/kafka-shared-msk/cbc` as comment.

File: prod-aws/kafka-shared-msk/cbc/cbc.tf [hunk 1]
```diff
@@ -46,3 +52,11 @@ module "cbc_events_indexer" {
   consume_groups   = ["cbc.cbc-events-indexer"]
   cert_common_name = "cbc/cbc-events-indexer"
 }
+
+module "cbc_account_events_relay" {
+  source           = "../../../modules/tls-app"
+  produce_topics   = []
+  consume_topics   = ["account-identity.public.account.events"]
+  consume_groups   = ["account-identity.cbc-fraud-detection-consumer-v1"]
```
Comment (LukeG07) [inline prod-aws/kafka-shared-msk/cbc/cbc.tf:60]:
```suggestion
  consume_groups   = ["account-identity.cbc-account-events-relay-v2"]
```

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 2]
```diff
@@ -1297,6 +1301,8 @@ module "cbc_legacy_account_eqdb_loader" {
 module "cbc_account_events_relay" {
   source           = "../../../modules/tls-app"
   produce_topics   = [kafka_topic.legacy_account_events_v2.name]
+  consume_topics   = ["account-identity.public.account.events"]
+  consume_groups   = ["account-identity.cbc-fraud-detection-consumer-v1"]
```
Comment (LukeG07) [inline dev-aws/kafka-shared-msk/cbc/cbc.tf:1305]:
```suggestion
  consume_groups   = ["account-identity.cbc-account-events-relay-v2"]
```

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: prod-aws
Module: pubsub/kafka-shared-msk-cbc
Path: prod-aws/kafka-shared-msk/cbc
Commit ID: 9bb5fb5d7c5d50e9ee6e3ec62c77bc72191eb667
Requested At: 2024-10-09T14:15:16Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan prod-aws/kafka-shared-msk/cbc` as comment.

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-cbc
Path: dev-aws/kafka-shared-msk/cbc
Commit ID: 864ad7835bd27f57a0641a4a8acc0c4a62ebaf65
Requested At: 2024-10-09T14:15:23Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/cbc` as comment.

**Review by LukeG07 — APPROVED** [review]

**Pull request merged**